### DISCUSSION TITLE
Как открыть Ваш проект в MicrosoftVisualStudio

### DISCUSSION THREAD

**Discussion by APDATe** [FreeRDP/FreeRDP#8363] [Q&A]
Добрый день!

пытался открыть проект в MicrosoftVisualStudio но не выходит (

не могли бы бросить в меня ссылкой как открывать проект в MicrosoftVisualStudio или какой IDE использовать?

я просто привык к MicrosoftVisualStudio

**Comment by hardening** [comment]
Ask questions in English please
